src/commands/login.rs

```diff
@@ -37,6 +39,12 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
             .unwrap_or("<unknown>")
     );
 
+    // It's not currently possible to easily mock I/O with `trycmd`,
+    // but verifying that `execute` is not panicking, is good enough for now.
+    if env::var("TEST").is_ok() {
```

Comment:
`TEST` is also pretty generalized, maybe we choose a more specific env variable name?

We should also debug log so it's clear why we are skipping the rest of `execute`.

This also applies to the rest of the usages of `env::var("TEST")`

Reply:
Updated.